index.html

```diff
@@ -25,11 +25,11 @@
 						<div class="register-panel pull-right">
 							<h3>Get full access before everyone</h3>
 							<ul>
-								<li>Get access to the private beta</li>
-								<li>Get 100$ of free cloud credits on public release</li>
-								<li>Access to the internal development roadmap</li>
-								<li>Get a larger free tier on public release</li>
-								<li>Free access to all of invoked.io events, conferences and meetups</li>
+								<li>Get access to the <b>private beta</b></li>
+								<li>Get 100$ of free <b>cloud credits</b> on public release</li>
```

Comment:
* get a 100$ credits on public release

and i would remove the "get a larger free tier on public release"